src/server/rpc/procedures/chart/node-gnuplot.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable quotes */
```

Comment:
why did you add this? I am not sure if this file justifies an exception to this rule...

Reply:
because I'm using the single quotes inside the strings thus I can't use em to surround sentences with the same character, but the rule doesn't detect that.